internal/pkg/workspace/workspace.go

```diff
@@ -462,18 +465,14 @@ func (ws *Workspace) ListDockerfiles() ([]string, error) {
 			if f.IsDir() {
 				continue
 			}
-
-			if f.Name() == dockerfileName {
-				directories = append(directories, wdFile.Name())
+			fname := f.Name()
+			if strings.Contains(strings.ToLower(fname), dockerfileName) {
+				path := wdFile.Name() + "/" + f.Name()
+				dockerfiles = append(dockerfiles, path)
 			}
 		}
 	}
-	sort.Strings(directories)
-	dockerfiles := make([]string, 0, len(directories))
-	for _, dir := range directories {
-		file := dir + "/" + dockerfileName
-		dockerfiles = append(dockerfiles, file)
-	}
```

Comment:
why do we decide to do it in the loop?

Reply:
Because `os.FileInfo` objects don't have the full path attached to them, just the base name. We have to move the logic to the loop so that we can append the full `subdir/NonStandardDockerfile` path to the output.